### PR TITLE
More changes for sr2020

### LINKED ISSUE #91: Broken links to Python tutorials
On the Python programming page (https://studentrobotics.org/docs/programming/python/), the links to "Byte of Python" and "Dive into Python" are both dead. There may be other broken links on the page; I haven't yet checked in detail.

### PR DESCRIPTION
**Pull request by TomWheal**
Few little fixes and some updates to vision and wire colours 

Resolves #91

### CODE REVIEW

**Comment by TomWheal** [comment]
This PR no longer contains rule related stuff

File: programming/python/index.md [hunk 1]
```diff
@@ -20,11 +20,10 @@ There are a number of tutorials out there which might help you to learn to progr
     there is a lot there and it may be a little overwhelming.
 
 * A number of tutorials for beginners are linked to from [here](http://wiki.python.org/moin/BeginnersGuide/NonProgrammers).
-    [Byte of Python](http://www.swaroopch.com/notes/Python_en:Functions) seems like a good one.
+    [Byte of Python](https://python.swaroopch.com/) seems like a good one.
```
Comment (RealOrangeOne) [inline programming/python/index.md]: It looks like this guide now defaults to Python 3, which may be confusing for some. Maybe it's worth simply pointing at the existing "Whirlwind Tour" page we have (https://studentrobotics.org/docs/tutorials/python)?

Comment (TomWheal) [reply programming/python/index.md]: Thats weird it defaulted to python 2.7 for me, ill have another look tomorrow

**Comment by kierdavis** [comment]
Resolves #91 (commenting so that github will automatically close the issue when this gets merged).

**Comment by TomWheal** [comment]
Fixes #91

File: programming/python/index.md [hunk 2]
```diff
@@ -20,11 +20,10 @@ There are a number of tutorials out there which might help you to learn to progr
     there is a lot there and it may be a little overwhelming.
 
 * A number of tutorials for beginners are linked to from [here](http://wiki.python.org/moin/BeginnersGuide/NonProgrammers).
-    [Byte of Python](http://www.swaroopch.com/notes/Python_en:Functions) seems like a good one.
+    [Wikibooks Tutorial](https://en.wikibooks.org/wiki/Non-Programmer%27s_Tutorial_for_Python_2.6) seems like a good one.
```
Comment (PeterJCLaw) [inline programming/python/index.md]: This tutorial claims to be for Python 2.6; that's really rather old at this point (and IIRC missing many of the features of Python 2.7). Are there other tutorials we could link to instead? (or maybe put this one a little lower down?)

File: kit/assembly.md [hunk 3]
```diff
@@ -35,10 +35,10 @@ Board                                   | Power
 [Ruggeduino](/docs/kit/ruggeduino)      | via USB
 [Servo Board](/docs/kit/servo_board)    | 12V
 
-In order to connect the Motor and Servo Boards to the Power Board,
- you will need to create a power cables.
-This should be done using the provided _power wire_ (this is the thicker wire provided in the kit)
- and a pair of green CamCon connectors.
+In order to connect the Motor and Servo Boards to the Power Board, you will need to create a power cables.
```
Comment (PeterJCLaw) [inline kit/assembly.md]:
```suggestion
In order to connect the Motor and Servo Boards to the Power Board, you will need to create some power cables.
```

**Comment by TomWheal** [comment]
@RealOrangeOne @PeterJCLaw @kierdavis can we get this merged?

**Pull request merged**